### PR TITLE
Some documentation updates

### PR DESCRIPTION
**Pull request by lee-dohm**
Saw these while looking around the documentation

### CODE REVIEW

**Comment by maxbrunsfeld** [comment]
:+1:

File: docs/creating-a-package.md [hunk 1]
```diff
@@ -83,9 +83,9 @@ module's `activate` method so you can restore your view to where the user left
 off.
 
 - `deactivate()`: This **optional** method is called when the window is shutting
-down. If your package is watching any files or holding external resources in any
-other way, release them here. If you're just subscribing to things on window,
-you don't need to worry because that's getting torn down anyway.
+down, or when your package is being updated or disabled. If your package is
+watching any files, holding external resources, providing commands or subscribing
+to events release them here.
```
Comment (maxbrunsfeld) [inline docs/creating-a-package.md]: I think there should be a comma before "release them here".

**Comment by lee-dohm** [comment]
Thanks @maxbrunsfeld! Rebased on the latest `master`.

**Comment by maxbrunsfeld** [comment]
Thanks @lee-dohm!

**Pull request merged**